src/bin/stratis-min/stratis-min.rs

```diff
@@ -103,6 +103,9 @@ fn parse_args() -> Command {
                 Command::new("has-passphrase")
                     .arg(Arg::new("name").long("name").num_args(0))
                     .arg(Arg::new("id").required(true)),
+                Command::new("is-bound")
```

Comment:
Please add a test for the new command in tests/stratis_min.rs, probably similar to ```stratis_min_pool_is_encrypted``` test.

Reply:
I'm going to hold off on this until you've gotten back to me about what you want written differently in that test.